### PR TITLE
documentation of mlan-ui project

### PR DESCRIPTION
**Pull request by devAmlan**
This is documentation of my project. I have added all components that I have made so far. Please review my PR.

### CODE REVIEW

**Comment by devAmlan** [comment]
preview of the branch -> https://preview-docs-mlanui.netlify.app/library/docs/docs.html

File: library/docs/docs.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+   display: flex;
+   flex-wrap: wrap;
+   padding-left: 1rem;
+   padding-right: 1rem;
+   margin: 1rem auto;
+}
+
+aside {
+   flex-basis:200px;
+   background-color:#14B8A6;
```
Comment (DivyanshSareen) [inline library/docs/docs.css]: var for color

File: library/docs/docs.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    text-decoration:none;
+    color:#FFFFFF;
+    display:flex;
+    justify-content:center;
+    align-items:center;
+    font-weight:800;
+  }
+
+.component-area{
+   margin:20px;
```
Comment (DivyanshSareen) [inline library/docs/docs.css]: can use relative units here

File: library/docs/docs.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+   aside {
+    display:none;
+    }
+    main{
+      width:250px;
+    }
+    .component-area{
+      margin:5px;
+      width:250px;
+      padding:10px;
```
Comment (DivyanshSareen) [inline library/docs/docs.css]: can use relative units

**Review by DivyanshSareen — APPROVED** [review]
Code looks good, good to merge

File: library/docs/docs.css (hunk 1 above)
Comment (devAmlan) [reply library/docs/docs.css]: I will change it thanks divyansh

File: library/docs/docs.css (hunk 3 above)
Comment (devAmlan) [reply library/docs/docs.css]: I will change to rems , thanks divyansh

**Review by developer-anand — APPROVED** [review]
Approved

File: library/docs/docs.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+aside {
+   flex-basis:20rem;
+   background-color:var(--Turquoisecolor);
+   height:50rem;
+ }
+ main {
+   flex-grow: 1;
+ }
```
Comment (bhushannaikaroskar) [inline library/docs/docs.css]: Hey amlan can understand that you are using this for docs but its better to use class files. Also in main tag if you are just using flex-grow please separate it in utility class.

File: library/docs/docs.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    justify-content:center;
+    align-items:center;
+    font-weight:800;
+  }
+
+.component-area{
+   margin:2rem;
+   border-radius:7px;
+   box-shadow:0 4px 8px 0 var(--shadow-color1), 0 6px 20px 0 var(--shadow-color2);
+   padding:1rem;
```
Comment (bhushannaikaroskar) [inline library/docs/docs.css]: Hey you have may have missed this, but please use rem here before merging

File: library/docs/docs.css [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+}
+.component-area .heading{
+   font-size:2.5rem;
+   text-align:left;
+}
+.component-description{
+  font-size:1.8rem;
+  margin:1rem 0px;
+  text-align:left;
```
Comment (bhushannaikaroskar) [inline library/docs/docs.css]: you can use 0px not an issue I can understand that may be your coding style. But its better to use 0 as value rather that 0px. Also there are places where you have used 0 and here you used 0px so its good to follow one practice throughout. You have use 0 at most of places I would recommend you to go ahead using 0.

File: library/docs/docs.css [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+   flex-wrap: wrap;
+   padding-left: 1rem;
+   padding-right: 1rem;
+   margin: 1rem auto;
+}
+
+aside {
+   flex-basis:20rem;
+   background-color:var(--Turquoisecolor);
+   height:50rem;
```
Comment (saurabhan) [inline library/docs/docs.css:12]: add some padding so that the content is not sticking to the box

**Review by bhushannaikaroskar — APPROVED** [review]
Hey Amlan reviewed your code good class names. Have give some small changes please do consider those before merging I am approving as of now.

**Review by saurabhan — APPROVED** [review]
looks good, approved

File: library/docs/docs.css (hunk 7 above)
Comment (bhushannaikaroskar) [reply library/docs/docs.css:12]: Amlan you can use prettier code extension for vscode, it auto-formats these small spacing which we generally miss out.

File: library/docs/docs.css (hunk 4 above)
Comment (devAmlan) [reply library/docs/docs.css]: ok i will change it thanks bhushan

File: library/docs/docs.css (hunk 6 above)
Comment (devAmlan) [reply library/docs/docs.css]: yeah bhushan i will use 0 . thanks for pointing it out.

File: library/docs/docs.css (hunk 7 above)
Comment (devAmlan) [reply library/docs/docs.css:12]: thanks @saurabhan  i will add padding here.

Comment (devAmlan) [reply library/docs/docs.css:12]: thanks for the suggestion @bhushannaikaroskar I will use this extension.

**Pull request merged**